README.md

```diff
@@ -603,17 +603,21 @@ python -m twine upload .\dist\*
 
 | Replace | With | Notes
 |:--------|:-----|:----------------------------
+| `QFont().fromString(...)` | `QtCompat.QFont.fromString(font, ...)` | Qt6 adds extra data to font strings and changes weight if you want to pass a font string from Qt6 to Qt4/5 use this
```

Comment:
Makes it possible to take a font string saved in Qt6 easily be loaded in Qt5. By default it would log a warning and not load the font.

```py
>>> fs = "Arial,7,-1,5,400,0,0,0,0,0,0,0,0,0,0,1"
>>> from PySide2.QtGui import QFont
>>> f = QFont()
>>> f.fromString(fs)
QFont::fromString: Invalid description 'Arial,7,-1,5,400,0,0,0,0,0,0,0,0,0,0,1'
False
>>> f
<PySide2.QtGui.QFont(,-1,-1,5,50,0,0,0,0,0) at 0x000001196C664480>
>>> from PyQt5.QtGui import QFont
>>> f = QFont()
>>> f.fromString(fs)
QFont::fromString: Invalid description 'Arial,7,-1,5,400,0,0,0,0,0,0,0,0,0,0,1'
False
>>> f
<PyQt5.QtGui.QFont object at 0x00000119698EFD80>
```